pkg/webhooks/webhookpatch.go

```diff
@@ -84,6 +84,17 @@ func NewWebhookCertPatcher(
 	return p, nil
 }
 
+func newWebhookPatcherQueue(reconciler controllers.ReconcilerFn) controllers.Queue {
+	return controllers.NewQueue("webhook patcher",
+		controllers.WithReconciler(reconciler),
+		// Try first few(5) retries quickly so that we can detect true conflicts by multiple Istiod instances fast.
+		// If there is a conflict beyond this, it means Istiods are seeing different ca certs and are in inconsistent
+		// state for longer duration. Slowdown the retries, so that we do not overload kube api server and etcd.
+		controllers.WithRateLimiter(workqueue.NewItemFastSlowRateLimiter(100*time.Millisecond, 1*time.Minute, 5)),
+		// Webhook patching has to be retried forever. But the retries would be rate limited.
+		controllers.RetryForever())
```

Comment:
Why not pass a maxint

Reply:
I can use that. This is to just make this clear that it is retried for ever.